__tests__/auth.js

```diff
@@ -55,7 +77,15 @@ describe('Auth', () => {
 
     await expect(jiraAddon.auth(req, {
       loadCredentials: () => jiraPayload
-    })).rejects.toThrow('Invalid signature')
+    })).rejects.toMatchError(
+      new AuthError(
+        'Invalid signature',
+        'INVALID_SIGNATURE',
+        new Error(
+          'Signature verification failed for input: eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJqaXJhLWNsaWVudC1rZXkifQ with method sha256'
+        )
```

Comment:
Not sure if it's a good idea to pass the original error object in this case as it reveals the token. But it's on the consumer, right? 🤔